scipy/stats/stats.py

```diff
@@ -3004,16 +3004,18 @@ def pearsonr(x, y):
 
     .. math::
 
-        r_{pb} = \frac{\sum (x - m_x) (y - m_y)}
-                      {\sqrt{\sum (x - m_x)^2 \sum (y - m_y)^2}}
+        r_{pb} = \frac{\sum_{i=1}^N (x_i - m_x) (y_i - m_y)}
+                {\sqrt{\sum_{i=1}^N (x_i - m_x)^2 \sum_{i=1}(y_i - m_y)^2}}
```

Comment:
^N missing in second sum

Reply:
looks like it missed  the +`^N` to auto-resolve-resolving.